.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+	"cSpell.words": ["recalc"]
```

Comment:
I think this needs to be a local only config for you

Reply:
true my fault.